server/models/Notification.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  photo: {
+    type: String,
+    default: "",
+  },
+  type: {
+    type: String,
+    enum: ["booking", "payment", "system"],
+    lowercase: true,
+    required: true,
+  },
+  link: {
+    type: String,
+    lowercase: true,
+    required: true,
+  },
+  status: {
+    type: String,
+    enum: ["read", "unread"],
+    default: "unread",
+    lowercase: true,
+    required: true,
+  },
+  date: {
```

Comment:
[Mongoose schemas have a timestamps option](https://mongoosejs.com/docs/guide.html#timestamps) that tells Mongoose to automatically manage createdAt and updatedAt properties on your documents